assets/js/components/Dashboard/ScreenDetail.tsx

```diff
@@ -14,6 +15,14 @@ const ScreenDetail = (props: ScreenDetailProps): JSX.Element => {
       ?.label;
   };
 
+  const isPaess = props.screens.some((screen) => screen.type === "pa_ess");
+
+  const getPaessRoute = () => {
+    return props.screens[0].station_code
+      ? props.screens[0].station_code.charAt(0).toLowerCase()
+      : "";
+  };
+
   const getScreenLocation = () =>
```

Comment:
This function needs to be changed to allow for getting the "location" of the countdown clocks. The designs show the route/mode as the location.

Reply:
Ah yeah I was wondering about that since an alternative solution would be to add locations data to the new json file for each of the PAESS screens. That sounds like a lot of annoying work for something that we could do fairly easily in this helper function though so I'll go ahead and add that here